go.mod

```diff
@@ -1,196 +1,223 @@
 module github.com/datachainlab/ethereum-ibc-relay-chain
 
-go 1.20
+go 1.21
+
+toolchain go1.22.2
```

Comment:
I am not familiar with go mod, but is this directive required since go 1.21? If yes, is there any reason why that `toolchain` minor version differs from the `go` directive one?

Reply:
Thank you.
As far as I understand, `toolchain` is optional and only takes effect when it's in the main module's go.mod.
Therefore I should delete this line.

done https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/40/commits/69993e978b4ea42aa4cfd51cbcba41ede798c14c